backend/src/db/migrate/store.js

```diff
@@ -0,0 +1,97 @@
+import { getDriver, getNeode } from '../../db/neo4j'
+
+class Store {
+  async init(next) {
+    const neode = getNeode()
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T14:01:03Z" title="Thursday, January 23rd 2020, 3:01:03 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -35,7 +35,7 @@ class Store {
     const session = driver.session()
     const readTxResultPromise = session.readTransaction(async txc => {
       const result = await txc.run(
-        'MATCH (migration:Migration) RETURN migration {.*} ORDER BY migration.timestamp DESC',
+        'MATCH (migration:Migration) RETURN migration {.*} ORDER BY migration.createdAt DESC',
```


One could argue we should save two attributes: `createdAt` and `migratedAt`

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T14:07:12Z" title="Thursday, January 23rd 2020, 3:07:12 pm +01:00">Jan 23, 2020</time>_

---


maybe, but I guess those will be the same cause the `Migration` is only created in the database when it is run for the first time... after that, since we use merge, it is not updated.
I thought about calling it `migratedAt`, and wouldn't be opposed to it, but thought that `createdAt` could also be ok...